app/javascript/components/patient/history/HistoryList.js

```diff
@@ -164,10 +160,14 @@ class HistoryList extends React.Component {
                 onChange={this.handleTypeFilterChange}
               />
             </Row>
-            {this.state.displayedHistories.map(histories => (
-              <History key={histories[0].id} versions={histories} current_user={this.props.current_user} authenticity_token={this.props.authenticity_token} />
-            ))}
-            <Row className="mx-3 mt-3 justify-content-end">
+            <section role="list" aria-label="History Items">
+              {this.state.displayedHistories.map((histories, index) => (
```

Comment:
Visually, this is clearly an easy to read list. However, with your eyes closed it's easy to not understand the context. By creating a `<section role="list"` the screen reader will inform the user that this a list of items, and read through them. It makes it much easier to understand (and doesnt visually change the styling at all). Again, try this on demo for a comparison.

Reply:
this is really cool